.github/workflows/mypy.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint
```

Comment:
Is there a reason why this is not another step in the lint.yml like spellcheck?
In the checks tab we now have 2 Lint workflows. See screenshot:
![image](https://user-images.githubusercontent.com/13337103/207472079-48ab7ef8-4577-4f6e-b621-e7e8ba8ad4db.png)

Reply:
makes sense, fixed - will merge once I confirm CI signal is green